sdk/agrifood/agrifood-farming-rest/src/farmBeats.ts

```diff
@@ -306,8 +306,10 @@ import {
   WeatherCreateDataDeleteJob202Response,
   WeatherCreateDataDeleteJobdefaultResponse,
 } from "./responses";
-import { getClient, ClientOptions, Client } from "@azure-rest/core-client";
+import { getClient, ClientOptions, Client, PathUncheckedResponse } from "@azure-rest/core-client";
 import { TokenCredential } from "@azure/core-auth";
+import { PagedAsyncIterableIterator } from "@azure/core-paging";
```

Comment:
I wonder if we should re-export this from `core-client-paging` for convenience to eliminate an extra dependency / import line?